cargo-near/src/commands/abi_command/abi.rs

```diff
@@ -93,6 +94,7 @@ pub(crate) fn generate_abi(
             ("CARGO_PROFILE_DEV_OPT_LEVEL", "0"),
             ("CARGO_PROFILE_DEV_DEBUG", "0"),
             ("CARGO_PROFILE_DEV_LTO", "off"),
+            (BUILD_RS_ABI_STEP_HINT_ENV_KEY, "true"),
```

Comment:
Do I see it right that it is not used anywhere? Let's remove dead code

Reply:
It is actually in use: https://github.com/dj8yfo/cargo-near/blob/ac64571dbe1246175e2f5ba64582dfd7b439a74f/cargo-near/src/build_extended/build_script.rs#L44C46-L47